src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1902,6 +1909,39 @@ private void StartAutoEvoPrediction()
 
         UpdateCompoundLastingTimes(compoundBalanceData, nightBalanceData, nominalStorage,
             specificStorages ?? throw new Exception("Special storages should have been calculated"));
+
+        // Handle process list
+        var processes = new List<TweakedProcess>();
+        var processStatistics = new List<ProcessSpeedInformation>();
+
+        ProcessSystem.ComputeActiveProcessList(editedMicrobeOrganelles, ref processes);
+
+        float consumptionProductionRatio = energyBalance.TotalConsumption / energyBalance.TotalProduction;
+
+        foreach (var process in processes!)
+        {
+            var singleProcess = ProcessSystem.CalculateProcessMaximumSpeed(process, biome, CompoundAmountType.Current);
+
+            if (consumptionProductionRatio < 1.0)
```

Comment:
What does this line do (could you add a comment)? Also the number should be `1.0f`.